test/spec/core/ElementRegistrySpec.js

```diff
@@ -206,6 +206,59 @@ describe('ElementRegistry', function() {
 
   });
 
+  describe('find', function() {
```

Comment:
Make sure you have _two new lines_ between any test-group (describe) and any test (it).